server/routes/__tests__/comments.test.js

```diff
@@ -3,26 +3,44 @@ const server = require('../../server')
 const db = require('../../db')
 
 import { objComment } from '~/test/fake-data'
+const { checkJwt } = require('../../utils')
 
 jest.mock('../../db')
+jest.mock('../../utils')
+
+beforeAll(() => {
+  jest.spyOn(console, 'log')
+  console.log.mockImplementation(() => {})
+  checkJwt.mockImplementation((req, res, next) => {
+    next()
+  })
+})
+
+afterAll(() => {
+  console.log.mockRestore()
+  jest.restoreAllMocks()
+})
 
 describe('POST /api/comments', () => {
   it('adding a comment hits the db', () => {
     db.insertComment.mockReturnValue(Promise.resolve(objComment))
+
     return request(server)
       .post('/api/comments')
       .send(objComment)
       .then((res) => {
         expect(res.status).toBe(200)
```

Comment:
expecting the status to be 200 is agood start. I would also want to see some assertions that `db.insertComment` was called with the correct arguments